### ISSUE TITLE
[Breadcrumb][High Contrast] Border broken when breadcrumb item is having href property

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#2781]
### Bug Report

- __Package version(s)__: 4.35.1002
- __Browser and OS versions__: Edge

#### Describe the issue:

In high contrast mode, if breadcrumb item is having href, then border appears broken.
The issue is because of the breadcrumb item with href is taking more space. For fixing this "chevron" should have more margin left.

#### Actual behavior:

Border appears broken when focused
![image](https://user-images.githubusercontent.com/6390909/30199421-4a393d72-9490-11e7-9812-8fc5a1426d79.png)

#### Expected behavior:

Border should be complete

### If applicable, please provide a codepen repro:

Official fabric site:
https://dev.office.com/fabric#/components/breadcrumb
